internal/sourcer/source.go

```diff
@@ -0,0 +1,105 @@
+package sourcer
+
+import (
+	"context"
+	"fmt"
+
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	registryClient "github.com/operator-framework/operator-registry/pkg/client"
+	utilerror "k8s.io/apimachinery/pkg/util/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	platformv1alpha1 "github.com/timflannagan/platform-operators/api/v1alpha1"
+)
+
+const (
+	channelName = "4.12"
+)
+
+type Bundle struct {
+	Version  string
+	Image    string
+	Replaces string
+	Skips    []string
+}
+
+func (b Bundle) String() string {
+	return fmt.Sprintf("Version: %s; Image: %s; Repliaces; %s", b.Version, b.Image, b.Replaces)
```

Comment:
```suggestion
	return fmt.Sprintf("Version: %s; Image: %s; Replaces; %s", b.Version, b.Image, b.Replaces)
```

Reply:
Whoops - this was a debug artifact, but I'll keep it in anyways. Updated.